ui/src/components/References/GroupReference.tsx

```diff
@@ -219,12 +217,14 @@ function GroupReference({
             ) : null}
             {status === 'loading' ? (
               <div className="flex items-center space-x-2">
-                <span className="text-gray-400">Joining...</span>
+                <span className="text-sm font-semibold text-gray-400">
+                  Joining...
+                </span>
                 <LoadingSpinner />
               </div>
             ) : (
               <button
-                className="small-button ml-3 whitespace-nowrap bg-blue-softer text-blue dark:text-black"
```

Comment:
without the margin here, the buttons look too close together imo. Was removing it intentional?